library/alloc/src/collections/vec_deque/mod.rs

```diff
@@ -713,7 +714,8 @@ impl<T, A: Allocator> VecDeque<T, A> {
     /// assert_eq!(buf[1], 7);
     /// ```
     #[stable(feature = "rust1", since = "1.0.0")]
-    pub fn get_mut(&mut self, index: usize) -> Option<&mut T> {
+    #[rustc_const_stable(feature = "const_vecdeque_methods", since = "1.90.0")]
```

Comment:
This would make these immediately stable, which we don't want to do. Please change this to `rustc_const_unstable` and create a new tracking issue.